### ISSUE TITLE
Console Spam

### ISSUE THREAD

**Issue by GiansCode** [rlf/uSkyBlock#456]
Whenever a new player joins the server, this message constantly spams the console until they create an island.

[13:21:01] [Craft Scheduler Thread - 3/INFO]: [uSkyBlock] Creating new player config!
[13:21:01] [Craft Scheduler Thread - 3/INFO]: [uSkyBlock] Creating player config Paths!
[13:21:01] [Craft Scheduler Thread - 23/INFO]: [uSkyBlock] Creating new player config!
[13:21:01] [Craft Scheduler Thread - 23/INFO]: [uSkyBlock] Creating player config Paths!
[13:21:05] [Craft Scheduler Thread - 3/INFO]: [uSkyBlock] Creating new player config!
[13:21:05] [Craft Scheduler Thread - 3/INFO]: [uSkyBlock] Creating player config Paths!
[13:21:05] [Craft Scheduler Thread - 8/INFO]: [uSkyBlock] Creating new player config!
[13:21:05] [Craft Scheduler Thread - 8/INFO]: [uSkyBlock] Creating player config Paths!

I'm not sure if it's something I did wrong

**Comment by dutchy1001** [comment]
Not sure why this would happen, can you do the command "/usb version" and post those details here plz.

**Comment by GiansCode** [comment]
[10:42:24 INFO]: Name: uSkyBlock
[10:42:24 INFO]: Version: 2.3-HF3
[10:42:24 INFO]: Description: Ultimate SkyBlock v2.3-HF3-9fc81a-dev
[10:42:24 INFO]: ------------------------------
[10:42:24 INFO]: Server: CraftBukkit git-Spigot-0359393-9a17f01 (MC: 1.8.8)
[10:42:24 INFO]: ------------------------------
[10:42:24 INFO]: Name: Vault
[10:42:24 INFO]: Version: 1.5.6-b49
[10:42:24 INFO]: ------------------------------
[10:42:24 INFO]: Name: WorldEdit
[10:42:24 INFO]: Version: 6.1.1-SNAPSHOT;3394-9a65bdb7
[10:42:24 INFO]: ------------------------------
[10:42:24 INFO]: Name: WorldGuard
[10:42:24 INFO]: Version: 6.1.1-SNAPSHOT.1628-e28b236
[10:42:24 INFO]: ------------------------------

**Comment by dutchy1001** [comment]
Upon joining the server as a new player it is correct you see this message, but only once. I have tried to reproduce this with the only difference that WG is build 1629 instead of yours 1628, but no spam of any sort.
I have a feeling that it is caused by an other plugin but that could be tricky to find out.
Are you on a host or run the server from home? (if on a host could there be some write protection??)

If all fails you could try the latest [pre-release](https://github.com/rlf/uSkyBlock/releases/tag/v2.3-HF7j)? These have still some bugs so be cautious and backup the skyworld and the plugins folder to be save.

**Comment by csorrows** [comment]
I have setup many servers with many different mixes of versions of the above. I have never seen this issue. With the exception of WE/WG 6.1.1. @dutchy1001, does the plugin retry if it doesn't see the changes saved to the file? Does it check the file contents after the change to verify? If so, it might somehow be some kind of file permissions issue.

**Comment by dutchy1001** [comment]
I'm not sure if it retries to add the files if it fails the first time, one of our devs could only answer that. As I never seen this on my servers I made (and I made plenty too) It just happens that in a previous issue it started to spam about the topten generation, which I couldnt reproduce either. I still think it might be caused by an other plugin or file system write protections.

**Comment by dutchy1001** [comment]
As this isn't happning in the pre-releases I will close this.
if the spam re appears you can always reopen or create a new issue.
